client/battle/CBattleInterfaceClasses.h

```diff
@@ -51,9 +52,10 @@ class CBattleConsole : public CIntObject
 class CBattleHero : public CIntObject
 {
 	void switchToNextPhase();
+
 public:
 	bool flip; //false if it's attacking hero, true otherwise
-	CDefHandler *dh, *flag; //animation and flag
+	CDefHandler * dh, * flag; //animation and flag

```

Comment:
We'll certainly have them. We talked about that with @alexvins privately and there is certainly very few cases where you have legit reason to use comma in declaration.

I commented on it to Tow: https://github.com/vcmi/vcmi/pull/347#discussion_r127613322